parse.go

```diff
@@ -183,7 +183,7 @@ func (p *parser) value(it item) (interface{}, tomlType) {
 				it.val)
 		}
 		val := strings.Replace(it.val, "_", "", -1)
-		num, err := strconv.ParseInt(val, 10, 64)
+		num, err := strconv.ParseInt(val, 0, 64)
```

Comment:
Since you changed the base to `0` you can remove that `strings.Replace` the line above this. `ParseInt` permits underscores if the base is 0 (and *only* of it's 0, for some reason, I think they did that for compatibility when the new literals were added in Go 1.13).

Reply:
ack